configs/config.yaml.example

```diff
@@ -101,3 +101,10 @@ mocks:
 
 # Replace kafka with postgres tasking system for introspection
 new_tasking_system: True
+
+# Admin task API
+admin_tasks:
+  # Enables API to view tasks across all accounts/organizations
+  enabled: False
+  # IDs for accounts that have access to this API
+  allowed_accounts: ["99999999"]
```

Comment:
we chatted recently about this, but i could see this easily fitting under the features area of the config here: https://github.com/content-services/content-sources-backend/pull/309/files#diff-c2315d27f02242f251151819c7cd0315404cd7bbaa0705c46a0755e5736f6428R102-R106

a bonus is that it should automatically expose it via the /features  api, so the UI would be able to fetch the features and see if it should expose the 'admin' tab to the user